### PR TITLE
Name Suggestion

### PR DESCRIPTION
**Pull request by ZELFs**
The Peach Consortium
^
This is a name suggestion. I like the sound of it, it's official but cute. I like that it can be written like this The :peach: Consortium and I like that it also enables us to inofficially still be called buttsortium. I like the name buttsortium because it's fun and makes things a bit lighter. I don't want this space to only feel like work (I like playfulness!) while it would simultaniously be great if the name can be a good official face outwards.

I still think we can use the domain name André registered for this year and then switch once the time comes, or set up dual link to a site with two domains.

### CODE REVIEW

**Comment by Powersource** [comment]
w

File: reasoning.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# The Reasoning Behind our Rules
+# Reasoning Behind The Peach Consortiums Rules
```
Comment (Powersource) [inline reasoning.md]: "Reasoning Behind the Rules of The Peach Consortium"

**Comment by elavoie** [comment]
If we are to apply for funding at the EU level, being playful may work against us. But we could always have two names (and two urls!), one playful for public relations with users and community members, and one "serious" for relations with institutions.

**Comment by staltz** [comment]
PEACH: Primary European Association of Community Hackers

**Comment by staltz** [comment]
PEACH: People-first European Association of Computational Humanity

**Comment by ZELFs** [comment]
P.E.A.C.H Consortium

**Comment by staltz** [comment]
This proposal (as per Mumble meeting) is to: **change the name of the EU consortium to "_The Peach Consortium_"**.

Vote results:
@staltz: y
@arj03: y
@elavoie : y
@Powersource : w
@ZELFs: y
@keks: pending
@cblgh: pending (optional)

**Comment by arj03** [comment]
@keks did I remember that you said w to this proposal?

**Comment by keks** [comment]
Moving from "Scuttlebutt Europe" to "Peach"/"PEACH"/"P.E.A.C.H." is if-need-be-but-not-excited.

I don't think we should have "Consortium" in our name, because I don't think it  gives a wrong impression of what we are. Currently, the only contender is "Association", but I'm not sure that is very good either. Maybe Platform?

**Comment by staltz** [comment]
@keks do you consider your comment to be a `!` or a `w`?

**Comment by keks** [comment]
!

**Comment by arj03** [comment]
Another suggestion: European SSB collective

From wikipedia: A collective is a group of entities that share or are motivated by at least one common issue or interest, or work together to achieve a common objective.

**Comment by cblgh** [comment]
dat's also thinking about moving from "foundation"/consortium to "collective" fwiw :)

**Comment by elavoie** [comment]
'European SSB collective' sounds great to me! Better than 'association' or 'consortium'.

From [Cambridge Dictionary](https://dictionary.cambridge.org/us/dictionary/english/collective): *an organization or business that is owned and controlled by the people who work in it*. 

This certainly emphasises the *democratic* decision-making that I care about. And it echoes the "It's time to build our own Internet" of the main SSB website. I also think it is important to keep emphasising that SSB is built first and foremost for the people that *actively participate* in its development, which 'collective' does. Otherwise, we risk opening ourselves to opportunists trying to co-opt the developers and community for their own aspirations.

**Comment by staltz** [comment]
> European SSB Collective

### `y`

**Comment by cblgh** [comment]
oh and in case it wasn't clear: i also vote `y` on this

**Comment by cblgh** [comment]
**European SSB Collective** is the name of this group/organization/thingerino!! (passed with 6 `y`, 1 `w`)

**Comment by cblgh** [comment]
i swapped the name from peach collective to European SSB Collective and im just gonna merge it

**Comment by Powersource** [comment]
loool just for posterity i'm gonna link the document we had with name ideas https://hackmd.io/CjUGbxEdQfuNWl4HCKvEPw

including stuff like

```
Peaches Agglomerated

The League of Butts

Eurocrabs
```

**Pull request merged**